build/hivedscheduler/go-build.sh

```diff
@@ -37,6 +38,9 @@ rm -rf ${DIST_DIR}
 mkdir -p ${DIST_DIR}
 
 go build -o ${DIST_DIR}/hivedscheduler cmd/hivedscheduler/*
+if [ ! -z ${TEST_FLAG} ] && [ ${TEST_FLAG} == "test=true" ]; then
+    go test ./...
+fi
```

Comment:
-race?

Reply:
it's impossible in linux/arm64 alpine until Go 1.15